sdk/storage/azure-storage-blob/azure/storage/blob/aio/__init__.py

```diff
@@ -67,7 +72,10 @@ async def upload_blob_to_url(
     :rtype: dict(str, Any)
     """
     async with BlobClient.from_blob_url(blob_url, credential=credential) as client:
-        return await client.upload_blob(data=data, blob_type=BlobType.BlockBlob, **kwargs)
+        return await client.upload_blob(
+            data=data,
+            blob_type=BlobType.BlockBlob,
+            **kwargs)  # type: ignore [no-any-return]
```

Comment:
azure\storage\blob\aio\__init__.py:75: error: Returning Any from function declared to return "Dict[str, Any]"  [no-any-return]

Reply:
Fixed with cast like in sync